dnf-behave-tests/dnf/remove-pkgspec.feature

```diff
@@ -29,6 +29,14 @@ Scenario Outline: Remove an RPM by <pkgspec-type>
         | remove-unused | glibc-all-langpacks-0:2.28-9.fc29.x86_64  |
     And package state is
         | package | reason | from_repo |
+    And transaction items for transaction "last" are
+        | action | package                                  | reason | repository |
+        | Remove | basesystem-0:11-6.fc29.noarch            | Clean  | @System    |
+        | Remove | filesystem-0:3.9-2.fc29.x86_64           | Clean  | @System    |
+        | Remove | glibc-0:2.28-9.fc29.x86_64               | User   | @System    |
+        | Remove | glibc-all-langpacks-0:2.28-9.fc29.x86_64 | Clean  | @System    |
+        | Remove | glibc-common-0:2.28-9.fc29.x86_64        | Clean  | @System    |
+        | Remove | setup-0:2.12.1-1.fc29.noarch             | Clean  | @System    |
```

Comment:
It is also a problem of dnf4 - it uses as repository for remove action always `@System`. Be honest it is not informative value at all.

Reply:
Maybe this could be changed to `from_repo` for outbound actions...? not the goal of this PR though.